### DISCUSSION TITLE
[Q] How do I set the background color of cell comments? I just see that you can set the text color

### DISCUSSION THREAD

**Discussion by Lin-Min** [exceljs/exceljs#2362] [Q&A]
## 💬 Questions and Help

Please do not use issues for making a questions, for this purposes much better is GitHub Discussions feature:
https://github.com/exceljs/exceljs/discussions/categories/q-a

### How to make a question?

1. First, try to find it by using search bar - anware what are you looking for was be done before.
2. Push the `New discussion` button.
3. Select `Q&A` category.

**Comment by zurmokeeper** [comment]
@Lin-Min Can you give a code example and excel screenshot or file example?
